draft-ietf-wish-whep.md

```diff
@@ -633,6 +633,25 @@ data: /DA8AAAAAAAAAP///wb+06ACpQAmAiRDVUVJAACcHX//AACky4AMEERJU0NZTVdGMDQ1MjAwME
 ~~~~~
 {: title="scte35 example event"}
 
+### Events over WebRTC datachannel
+
+In some architectures sending events over a long polled HTTP request from a server that isn't the media server may be costly, such servers may support events over the webRTC datachannel.
+The protocol is identical to the above, with the following exceptions:
+
+- If both the SDP offer and the SDP answer contain an application section this indicates that Datachannel events are supported.
```

Comment:
how does the player specify which events he wants to receive? currently it is done in the POST request to create the server sent event url

```
POST /resource/213786HF/sse HTTP/1.1
Host: whep.example.com
Content-Type: application/json

["active","inactive","layers","reconnect","viewercount"]
```

Reply:
I assumed this POST would remain the same, only the long polling is moved. I definitely need to be more clear here though. !

Reply:
I don't think  we have to change the POST URI - it is just an example, not normative - but it might be clearer if we did.